SingularityExecutor/src/main/resources/logrotate.conf.hbs

```diff
@@ -36,6 +36,7 @@ notifempty
 
 {{#if extrasFiles}}
 {{#each extrasFiles}}{{{filename}}} {
+  {{#if logrotateFrequencyOverride }}{{{logrotateFrequencyOverride}}}{{/if}}
```

Comment:
We should double check if this will 'just work'. Not all versions of logrotate support `hourly` by default (Many are normally run on a daily cron, not hourly). If you look in `SingularityExecutorTaskLogManager.writeLogrotateFile` you'll notice that we additionally write an hourly cron for the specific logrotate conf file that needs it. We will likely need an additional check in that method to see if an additional file has an hourly rotate and possibly a separate cron file if so (the current cron uses the `-f` option to support hourly and force a logrotate run for those versions that do not support it)

Reply:
Additional note, enough OSs likely support hourly logrotate now that we could do away with this extra check if we verify that it still works as intended

Reply:
We have discussed this in depth, and more recent OSs do support it (and we have been supporting it at HubSpot for a while). We have [already updated] (https://git.hubteam.com/HubSpotProtected/puppet-deploy/pull/3869) the import statement to include https://github.com/rodjek/puppet-logrotate/blob/master/manifests/hourly.pp in our Mesos slave Puppet deploy. I have updated the code to put logrotate config files in /etc/logrotate.d/hourly. We need to monitor this in QA and make sure there are no averse performance penalties.

Reply:
We have discussed this at length and determined that it's ok to put the logrotate config files at `/etc/logrotate.d/hourly`. The Mesos slave running our `SingularityExecutor` needs to support hourly cron jobs at this directory. We have already internally updated our puppet deploy to enforce this config on mesos slaves.

Reply:
We have decided to support older logrotate versions and added code to Singularity to force a rotation hourly using the `-f` flag and a cron schedule. This implementation writes config files to two directories:
* `/etc/logrotate.d`
* `/etc/logrotate.d/hourly`